plugins/plugin-chart-table/src/controlPanel.tsx

```diff
@@ -21,45 +22,163 @@ import { t } from '@superset-ui/translation';
 import {
   formatSelectOptions,
   D3_TIME_FORMAT_OPTIONS,
+  ControlConfig,
   ColumnOption,
+  ControlStateMapping,
+  ControlPanelConfig,
+  ControlPanelsContainerProps,
 } from '@superset-ui/chart-controls';
 import { validateNonEmpty } from '@superset-ui/validator';
 import { smartDateFormatter } from '@superset-ui/time-format';
 
 export const PAGE_SIZE_OPTIONS = formatSelectOptions<number>([[0, t('All')], 10, 20, 50, 100, 200]);
```

Comment:
This file is almost identical to `legacy-plugin-chart-table/src/controlPanel.tsx`, except updates on exporting `PAGE_SIZE_OPTIONS ` and default option values.